test/integration/pipeline/pipeline_test.cpp

```diff
@@ -98,3 +107,53 @@ TEST(PipelineIntegrationTest, SendTx) {
       .checkBlock(checkBlock)
       .done();
 }
+
+/**
+ * @given some user
+ * @when sending sample AddAssetQuantity transactions to the ledger
+ * @then receive STATELESS_VALIDATION_SUCCESS status on that transactions,
+ * all transactions are passed to proposal and does not appear in block
+ */
+TEST(PipelineIntegrationTest, SendTxSequence) {
+  size_t tx_size = 5;
+
+  shared_model::interface::types::SharedTxsCollectionType txs;
+
+  for (size_t i = 0; i < tx_size; i++) {
+    auto &&tx = prepareAddAssetQtyTransaction(iroha::time::now() + i);
```

Comment:
`iroha::time::now()` is better to be called once before the loop for simplicity.